_data/sites.json

```diff
@@ -9846,6 +9863,16 @@
         ]
     },
 
+    {
+        "name": "RedShelf",
+        "url": "https://solve.redshelf.com/hc/requests/new?ticket_form_id=1260804993770",
+        "difficulty": "hard",
+        "notes": "NOT FOR BRYTEWAVE ACCOUNTS.",
```

Comment:
Can you be a bit more specific? I think I didn't fully understand this. Also not using all caps.

Reply:
RedShelf has a lot of different subdomains, one of them being brytewave.

Except brytewave is operated by Follett, despite using the RedShelf account system (but Follett's ticket system doesn't list an option for brytewave).

Reply:
I see, thanks for explaining.